Resources/Prototypes/Entities/Objects/Weapons/Guns/HMGs/hmgs.yml

```diff
@@ -0,0 +1,77 @@
+- type: entity
+  name: BaseHMG
```

Comment:
give it an actual name

Reply:
It's based on "BaseLMG", it's just consistency.